src/schema/types/user.graphql

```diff
@@ -1,4 +1,9 @@
 type User {
   id: ID!
   name: String!
+  groups: [Group]
```

Comment:
Let's rename this. "groups", in the context of a User could have different meanings. In order to avoid ambiguity, let's calls this subscribedGroups.

We could also start to think a better name than just group. Maybe purchase group?